src/spec/validitygenerator.py

```diff
@@ -348,9 +348,9 @@ def createValidationLineForParameterIntroChunk(self, blockname, param, params, t
             if (lengths[0]) == 'null-terminated':
                 asciidoc += 'a null-terminated '
             elif (lengths[0]) == '1':
-                asciidoc += 'a pointer to '
+                asciidoc += 'a valid pointer to '
```

Comment:
> By current unspoken convention of the spec just the lack of "valid" prefix implies the object is exempt from validity. So this makes sense to me even from the consistency perspective.

Reasonable. Consistency should probably trump redundancy in this case.

> Besides, you are using "insider knowledge".

Perhaps :)

Like I said - I'm fine with this change, it's the rest of the WG I'm asking on behalf of since we changed it away from this in the past.